src/components/summary-list/index.md.njk

```diff
@@ -65,10 +67,14 @@ To remove borders on a single row, use the `govuk-summary-list__row--no-border`
 
 If you’re showing multiple summary lists on a page, you can show each list within a summary card. This lets you visually separate them and give each list a title and some actions.
 
-Use summary cards when you need to show: 
-- multiple summary lists of the same type, for example a series of interview appointments or application choices
+Use summary cards when you need to show either: 
+- multiple summary lists that repeat the same set of key labels, for example to describe a set of people or professional qualifications
 - actions that will apply to all the items in a list
 
+Summary cards are often used in case working systems, to help users quickly view information about a set of things and apply actions to them.
+
+However, summary cards may not always be appropriate for public-facing services as users might not easily understand the information the summary card is meant to show.
```

Comment:
Not sure about this line, seems pretty vague. I’m not sure the distinction is a public-facing service versus a case working system but more to do with the overall amount and/or complexity of the information being gathered.

Might this guidance be better suited to suggest that summary cards may not be appropriate for simple transactional services for which there is a single journey, or something along those lines? Or maybe this second line isn’t needed at all?

Reply:
```suggestion
```
Thanks Paul, good to hear this edit's on the right track!

I'd say let's remove the 'however' sentence — was trying to write something to discourage overuse, but does look like it'll quite work here.